### ISSUE TITLE
Fatal error: Uncaught Error: Class 'MyNamespace\stdClass' not found in DIR/property01abfc.php on line 33

### ISSUE THREAD

**Issue by gigi** [swaggest/php-code-builder#5]
Hi again!

Can'\t figure out how to work with if/else conditions.

Simple example

```json
{
  "type": "object",
  "properties": {
    "foo": {
      "type": "string"
    },
    "bar": {
      "type": "string"
    }
  },
  "if": {
    "properties": {
      "foo": {
        "enum": [
          "bar"
        ]
      }
    }
  },
  "then": {
    "required": [
      "bar"
    ]
  }
}
```

Generated class

```php
class property01abfc extends ClassStructure
{
    /** @var string */
    public $foo;

    /** @var string */
    public $bar;

    /**
     * @param Properties|static $properties
     * @param Schema $ownerSchema
     */
    public static function setUpProperties($properties, Schema $ownerSchema)
    {
        $properties->foo = Schema::string();
        $properties->bar = Schema::string();
        $ownerSchema->type = 'object';
        $ownerSchema->if = (object)array(
            'properties' => 
            stdClass::__set_state(array(
                 'foo' => 
                stdClass::__set_state(array(
                     'enum' => 
                    array(
                        0 => 'bar',
                    ),
                )),
            )),
        );
        $ownerSchema->then = (object)array(
            'required' => 
            array(
                0 => 'bar',
            ),
        );
    }

    /**
     * @return string
     * @codeCoverageIgnoreStart
     */
    public function getFoo()
    {
        return $this->foo;
    }
    /** @codeCoverageIgnoreEnd */

    /**
     * @param string $foo
     * @return $this
     * @codeCoverageIgnoreStart
     */
    public function setFoo($foo)
    {
        $this->foo = $foo;
        return $this;
    }
    /** @codeCoverageIgnoreEnd */

    /**
     * @return string
     * @codeCoverageIgnoreStart
     */
    public function getBar()
    {
        return $this->bar;
    }
    /** @codeCoverageIgnoreEnd */

    /**
     * @param string $bar
     * @return $this
     * @codeCoverageIgnoreStart
     */
    public function setBar($bar)
    {
        $this->bar = $bar;
        return $this;
    }
    /** @codeCoverageIgnoreEnd */
}
```

This gives error
```
Fatal error: Uncaught Error: Class 'MyNamespace\stdClass' not found in DIR/property01abfc.php on line 33

Error: Class 'MyNamespace\stdClass' not found in DIR/property01abfc.php on line 33
```

Why is `stdClass::__set_state` here? 
Thanx!

**Comment by vearutop** [comment]
The issue was fixed in [v0.2.4](https://github.com/swaggest/php-code-builder/releases/tag/v0.2.4), please upgrade.

Here are [example test](https://github.com/swaggest/php-code-builder/blob/master/tests/src/PHPUnit/Issues/Issue5Test.php#L48) and [generated code](https://github.com/swaggest/php-code-builder/tree/master/tests/src/Tmp/Issue5) for your case.

**Comment by gigi** [comment]
@vearutop seems to be working!
Thank You!) Moving on...)
